contribute-docs/how-to/cumulative-docs/reference.md

```diff
@@ -0,0 +1,24 @@
+# Quick reference
+
+The `applies_to` directive uses the following format:
+
+```
+<key>: <lifecycle> <version>
+```
+
+This page provides minimal reference information on the `applies_to` directive. For more detailed information, refer to [the applies_to syntax guide](https://elastic.github.io/docs-builder/syntax/applies).
+
+## key
+
+:::{include} /contribute-docs/_snippets/applies_to-key.md
+:::
+
+## lifecycle
+
+:::{include} /contribute-docs/_snippets/applies_to-lifecycle.md
+:::
+
+## version
+
+:::{include} /contribute-docs/_snippets/applies_to-version.md
+:::
```

Comment:
would like to see these more scannable. bulleted list very overwhelming (I realize you ported it that way)